src/UnityExtension/Assets/Editor/GitHub.Unity/UI/TreeControl.cs

```diff
@@ -383,7 +385,7 @@ private void ResetNodeIcons(TreeNode node)
             }
             else if (node.IsFolder)
             {
-                if (node.Level == 1)
+                if (node.Level == RootFolderIconLevel)
```

Comment:
Why was this switched from 1 to -1?

Reply:
You know what. It's a rudimentary fix for something that i'm already having to change.
I'm going to send a PR before this one that provides a correct solution instead.

Reply:
Sounds good